tests/conftest.py

```diff
@@ -38,8 +41,7 @@ def migrate_test_db() -> None:
 def db() -> Generator:
     """Return a connection to the test DB"""
     # Create the test DB enginge
-    ## This asserts that TESTING==True
-    assert os.getenv("TESTING", False)
+    assert os.getenv("TESTING") == "True"
```

Comment:
cc @stevenbenjamin — I've updated these checks to look like this now as that's the correct convention according to `pylint`. Environment vars can only be strings so returning a `bool` as the default arg is dangerous. Hopefully this syntax is clearer too.